webplugin/js/app/kommunicate.js

```diff
@@ -208,6 +208,14 @@ $applozic.extend(true, Kommunicate, {
         KommunicateUI.showChat();
         KommunicateUI.hideFaq();
     },
+    reloadWidget: function () {
+        ALStorage.clearAppHeaders();
```

Comment:
Did you check this functionality with the pre-chat lead collection?

Reply:
Yes. Working fine.